ui/app/models/mount-config.js

```diff
@@ -33,8 +33,11 @@ export default class MountConfigModel extends Model {
   auditNonHmacResponseKeys;
 
   @attr('mountVisibility', {
-    editType: 'boolean',
-    label: 'List method when unauthenticated',
+    label: 'Use as preferred UI login method',
+    editType: 'toggleButton',
+    helperTextEnabled: 'Shown as the default login method on the unauthenticated UI login endpoint.',
```

Comment:
This copy isn't quite accurate 🤔 I think some comments I left on the designs a while ago never made it to the final iteration. Can you update this and I'll make sure to confirm with design when I demo. 
```suggestion
    helperTextEnabled: 'This mount will be included in the unauthenticated UI login endpoint and display as a preferred login method.',
```